### PR TITLE
Simple check improvement

### PR DESCRIPTION
**Pull request by PaulZGit**
- Added Tests
- Added Graphical Visualisation

Todo: Adding Parameters

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -17,10 +17,10 @@
     # separator = args["sep"]
 
     detector = Detector(args['train'], args['test'], delimiter=args['sep'])
-    detector.add_checks(SimpleCheck())
```
Comment (phSch08) [inline main.py]: I think this file should not be in the repository. Can you delete it and write it on .gitignore?

File: morpheus.ipynb [hunk 2]
```diff
@@ -5,8 +5,8 @@
    "execution_count": 1,
```
Comment (phSch08) [inline morpheus.ipynb]: This file should not be in repository - please delete and put on .gitignore

File: shift_detector/checks/simple_check.py [hunk 3]
```diff
@@ -34,8 +35,7 @@ def relative_metric_difference(self, column, metric_name):
             return 0
         # TODO: think about comparison if base value is 0
```
Comment (phSch08) [inline shift_detector/checks/simple_check.py]: Have you thought about that? Please fix this TODO

File: shift_detector/checks/simple_check.py [hunk 4]
```diff
@@ -34,8 +35,7 @@ def relative_metric_difference(self, column, metric_name):
...
         if metric_in_df1 == 0:
-            logger.warning('column', column, '\t \t', metric_name,
-                           ': no comparison of distance possible, division by zero')
+            logger.warning('column {} \t \t {}: no comparison of distance possible, division by zero'.format(column, metric_name))
```
Comment (phSch08) [inline shift_detector/checks/simple_check.py]: Line is to long -> break after string

File: tests/checks/test_SimpleCheck.py [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+
+    def setUp(self):
+        sales1 = {'shift': ['A'] * 100, 'no_shift': ['C'] * 100}
+        sales2 = {'shift': ['B'] * 100, 'no_shift': ['C'] * 100}
+        self.df1 = DataFrame.from_dict(sales1)
+        self.df2 = DataFrame.from_dict(sales2)
+        self.store = Store(self.df1, self.df2)
+        self.check = SimpleCheck()
+
+    def test_init(self):
```
Comment (phSch08) [inline tests/checks/test_SimpleCheck.py]: This test makes no sense

File: tests/checks/test_SimpleCheck.py [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+        with self.subTest("Test shifted categorical columns"):
+            self.assertEqual(report.shifted_columns, ['shift'])
+            self.assertCountEqual(report.examined_columns, ['shift', 'no_shift'])
+            self.assertEqual(report.explanation['shift'], 'Attribute: A with Diff: 1.0\n')
+
+    def test_run_numerical(self):
+        with self.subTest("Test precalculation"):
+            numbers1 = {'cool_numbers': [1, 2, 3, 4]*10}
+            numbers2 = {'cool_numbers': [1, 2, 3, 6]*10}
+            self.df1 = DataFrame.from_dict(numbers1)
```
Comment (phSch08) [inline tests/checks/test_SimpleCheck.py]: Do this instead and delete lines 46, 47

```suggestion
            self.df1 = DataFrame({'cool_numbers': [1, 2, 3, 4] * 10})
```

File: main.py (hunk 1 above)
Comment (PaulZGit) [reply main.py]: it already is in the gitignore... changed its position, maybe it works now

File: morpheus.ipynb (hunk 2 above)
Comment (PaulZGit) [reply morpheus.ipynb]: wrong

Comment (PaulZGit) [reply morpheus.ipynb]: https://memegenerator.net/img/instances/73588682/wrong.jpg

File: shift_detector/checks/simple_check.py (hunk 3 above)
Comment (PaulZGit) [reply shift_detector/checks/simple_check.py]: yup, deprecated comment

File: shift_detector/checks/simple_check.py (hunk 4 above)
Comment (PaulZGit) [reply shift_detector/checks/simple_check.py]: changed

File: tests/checks/test_SimpleCheck.py (hunk 5 above)
Comment (PaulZGit) [reply tests/checks/test_SimpleCheck.py]: It did not, now it does not anymore

File: tests/checks/test_SimpleCheck.py (hunk 6 above)
Comment (PaulZGit) [reply tests/checks/test_SimpleCheck.py]: deprecated

**Pull request merged**